src/index.html

```diff
@@ -13,5 +13,142 @@
+        <label class="personal">
+          Surname: 
+          <input 
+            type="text" 
+            name="surname"  
+            autocomplete="off"
+          >  
+        </label>
+        <label class="personal">
+          Name:
+          <input 
+            type="text" 
+            name="name" 
+            autocomplete="off"
+          >  
+        </label>
+        <label class="personal">
+          How old are you?
+          <input
+            type="number" 
+            name="old" 
+            max="100" 
+            min="1" 
+            value="12"
+          >  
+        </label>
+        <label class="personal">
+          Full date of birth:
+          <input
+            type="date" 
+            name="birth"
+            >
+        </label>
+        <label class="personal">
+          I accept the term of the agreement
+          <input 
+            type="checkbox" 
+            name="accept" 
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="wrapper"> <legend>Registration</legend>
+          
+        <label class="registration">
+          E-mail:
+          <input 
+            type="email" 
+            name="email" 
+            placeholder="email@example.com"
+          >
+        </label>
+
+        <label class="registration">
+          Password:
+          <input 
+            type="password" 
+            name="password" 
+            minlength="8" 
+            maxlength="32" 
+          >
+        </label>
+      </fieldset>
+        
+      <fieldset class="wrapper"> <legend> An intresting facs about you</legend>
+          
+        <label class="facts">
+          Do you love cats? 
+          <input 
+            type="radio" 
+            name="cats"
+          > Yes
+          <input 
+            type="radio" 
+            name="cats"
+          > No
+        </label>
+            
+        <label class="facts">
+          What is your favorite color?
+            <input 
+              type="color" 
+              name="color"
+            >
+        </label>
+        <label class="facts">
+          What time do you go to bed?
+          <input 
+            type="time" 
+            name="time"
+          >
+        </label>
+        <label class="facts">
+          What is your favorite brand of cars?
+          <select name="car" multiple >
+            <option value="BMW" selected>BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">Lada</option>
+          </select>
+        </label >
+        <label class="facts">
+          How do you rate our work?
+          <input 
+            type="range"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="wrapper"> <legend>Additional information</legend>
+        <label class="additional">
+          Comments: 
+          <textarea
+            name="feedback" 
+            cols="30" 
+            rows="10"
+          ></textarea>
```

Comment:
```suggestion
          <textarea
            name="feedback" 
            cols="30" 
            rows="10"
        ></textarea>
```
Are you sure that it should be so big?
![image](https://user-images.githubusercontent.com/96409563/228154650-3465d886-af35-4faa-8dac-7beb8c79a8b7.png)